locations/spiders/big_o_tires_us.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+from scrapy import Spider
+from scrapy.downloadermiddlewares.retry import get_retry_request
+from scrapy.http import JsonRequest
+
+from locations.dict_parser import DictParser
+from locations.hours import OpeningHours
+from locations.searchable_points import open_searchable_points
+
+logger = logging.getLogger(__name__)
```

Comment:
Please use `self.logger` instead. I don't think this will get written out in existing logs.